src/Game.js

```diff
@@ -20,24 +23,35 @@ class Game {
     }
   }
 
+  turnChange() {
```

Comment:
Drobiazg, czasownik pierwszy 😄 
```suggestion
  changeTurn() {
```